Wrapping/Python/CMakeLists.txt

```diff
@@ -76,16 +76,27 @@ file( TO_NATIVE_PATH "${SimpleITK_PYTHON_PACKAGE_DIR}" SimpleITK_PYTHON_PACKAGE_
 
 set(SimpleITK_PYTHON_TEST_EXECUTABLE "${PYTHON_EXECUTABLE}" CACHE INTERNAL "Python executable for testing." FORCE )
 
+
+
+set(SimpleITK_Py_Files "__init__.py" "extra.py" )
+
 if(DEFINED SKBUILD)
   # Currently this installation
   install(FILES
-      ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK.py
-      ${CMAKE_CURRENT_SOURCE_DIR}/Packaging/__init__.py
+      ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK/SimpleITK.py
```

Comment:
${CMAKE_CURRENT_BINARY_DIR}/SimpleITK/SimpleITK.py 
change to:
${CMAKE_SWIG_OUTDIR}/SimpleITK.py